yarn.lock

```diff
@@ -18063,11 +18087,16 @@ typescript-transform-paths@^2.2.2:
   dependencies:
     minimatch "^3.0.4"
 
-typescript@4.1.3, typescript@^4.1.3, typescript@~4.0.2:
+typescript@^4.1.3:
   version "4.1.3"
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.1.3.tgz#519d582bd94cba0cf8934c7d8e8467e473f53bb7"
   integrity sha512-B3ZIOf1IKeH2ixgHhj6la6xdwR9QrLC5d1VKeCSY4tvkqhF2eqd9O7txNlS0PO3GrBAFIdr3L1ndNwteUbZLYg==
 
+typescript@~4.0.2:
```

Comment:
Another yarn resolution gotcha @peterp. Its because i did
```
cd packages/api-server
yarn add http-proxy-middleware
```
It ignored the resolutions at the top.

There must be a flag/fix for this right!?